src/components/GameSettings.js

```diff
@@ -0,0 +1,113 @@
+import { makeStyles } from "@material-ui/core/styles";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Radio from "@material-ui/core/Radio";
+import RadioGroup from "@material-ui/core/RadioGroup";
+import { Box } from "@material-ui/core";
+import Switch from "@material-ui/core/Switch";
+import Tooltip from "@material-ui/core/Tooltip";
+
+import firebase from "../firebase";
+
+const useStyles = makeStyles(() => ({
+  modeSelection: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-around",
+  },
+}));
+
+const normalTip = "Find 3 cards that form a Set";
+const setchainTip =
+  "In every next Set you have to use 1 card from the previously found Set";
+const ultrasetTip =
+  "Find 4 cards such that the first pair of them makes Set" +
+  "with the card that makes Set with the second pair";
+const hintTip =
+  "Practice mode where you can get hint for finding Sets." +
+  "Only available in private game with a single player. Not counted in total stats.";
+
+const modeInfo = {
+  normal: {
+    displayName: "Normal",
+    description: normalTip,
+  },
+  setchain: {
+    displayName: "Set-Chain",
+    description: setchainTip,
+  },
+  ultraset: {
+    displayName: "Ultra Set",
```

Comment:
In all other places we call it UltraSet, but here it's Ultra Set. Maybe stick to one?